public/index.html

```diff
@@ -4,11 +4,23 @@
   <link rel="stylesheet" type="text/css" href="styles/interface.css">
   <script src="/socket.io/socket.io.js"></script>
   <script src="scripts/main.js"></script>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
 </head>
 
 <body>
-  <input id="input" autocomplete="off" /><button onClick="sendMessage()">SEND</button>
+
+  
   <ul id='messages'></ul>
 </body>
+<footer>
+  <input class="input" id="input" autocomplete="off" action="sendMessage()" /><button class="button" id="button" onClick="sendMessage()">SEND</button>
```

Comment:
My general understanding is that class and I'd are both ways to describe a tag (class for an instance of a particular type, id for a particular tag you want to target), and CSS & JS let you select tag(s) based on tag type, class, or id.

Particularly, I imagine class is useful for divs/spans, because you might want to style divs with a class of foo very differently than divs with a class of bar.